bin/travis_analyzer

```diff
@@ -5,6 +5,7 @@ require "travis"
 
 require_relative "../lib/upgrade_analyzer/analyzer"
```

Comment:
Since you removed the UpgradeAnalyzer::Analyzer line. The `require` is not needed anymore.